cmd/policy/policy.go

```diff
@@ -0,0 +1,242 @@
+			Use:   "get <policyID>",
+			RunE: func(cmd *cobra.Command, args []string) error {
+				p, err := policy.NewClient(*policyBaseURL, config).GetPolicy(*ownerID, args[0])
+				if err != nil {
+					return fmt.Errorf("failed to get policy: %v", err)
+				}
+
+				enc := json.NewEncoder(cmd.OutOrStdout())
+				enc.SetIndent("", "  ")
+
+				if err := enc.Encode(p); err != nil {
+					return fmt.Errorf("failed to output policy in json format: %v", err)
+				}
+
+				return nil
+			},
+			Args:    cobra.ExactArgs(1),
+			Example: `policy get 60b7e1a5-c1d7-4422-b813-7a12d353d7c6 --owner-id 516425b2-e369-421b-838d-920e1f51b0f5`,
+		}
+		return cmd
+	}()
+
+	delete := func() *cobra.Command {
+		cmd := &cobra.Command{
+			Short: "Delete a policy",
+			Use:   "delete <policyID>",
+			RunE: func(cmd *cobra.Command, args []string) error {
+				err := policy.NewClient(*policyBaseURL, config).DeletePolicy(*ownerID, args[0])
+				if err != nil {
+					return fmt.Errorf("failed to delete policy: %v", err)
+				}
+				_, _ = io.WriteString(cmd.OutOrStdout(), "Deleted Successfully\n")
+				return nil
+			},
+			Args:    cobra.ExactArgs(1),
+			Example: `policy delete 60b7e1a5-c1d7-4422-b813-7a12d353d7c6 --owner-id 516425b2-e369-421b-838d-920e1f51b0f5`,
+		}
+
+		return cmd
+	}()
+
+	update := func() *cobra.Command {
+		var policyPath string
+		var active bool
+		var context string
+		var name string
+		var updateRequest policy.UpdateRequest
+
+		cmd := &cobra.Command{
+			Short: "Update a policy",
+			Use:   "update <policyID>",
+			RunE: func(cmd *cobra.Command, args []string) error {
+
+				if !(cmd.Flag("policy").Changed ||
+					cmd.Flag("active").Changed ||
+					cmd.Flag("context").Changed ||
+					cmd.Flag("name").Changed) {
+					return fmt.Errorf("one of policy, active, context, or name must be set")
+				}
+
+				if cmd.Flag("policy").Changed {
+					policyData, err := ioutil.ReadFile(policyPath)
+					if err != nil {
+						return fmt.Errorf("failed to read policy file: %w", err)
+					}
+
+					content := string(policyData)
+
+					updateRequest.Content = &content
+				}
+
+				client := policy.NewClient(*policyBaseURL, config)
+
+				if cmd.Flag("active").Changed {
+					updateRequest.Active = &active
+				}
+
+				if cmd.Flag("context").Changed {
+					updateRequest.Context = &context
+				}
+
+				if cmd.Flag("name").Changed {
+					updateRequest.Name = &name
+				}
+
+				result, err := client.UpdatePolicy(*ownerID, args[0], updateRequest)
+				if err != nil {
+					return fmt.Errorf("failed to update policy: %w", err)
+				}
+
+				enc := json.NewEncoder(cmd.OutOrStdout())
+				enc.SetIndent("", "  ")
+
+				if err := enc.Encode(result); err != nil {
+					return fmt.Errorf("failed to encode result to stdout: %w", err)
+				}
+
+				return nil
+			},
+			Args:    cobra.ExactArgs(1),
+			Example: `policy update e9e300d1-5bab-4704-b610-addbd6e03b0b --owner-id 462d67f8-b232-4da4-a7de-0c86dd667d3f --name policy_name --active --policy ./policy.rego`,
```

Comment:
Not a huge deal but these policy commands go against most of our input patterns found in other commands where it's usually `<command> <action> <name>` over name being set as a flag (examples: context, namespace). 

Might be a bit weird for our users for these commands to be different. As a user, personally I would find it a bit annoying to remember all the different syntaxes for all the different commands.

Reply:
In our case, name is a required input for `create` subcommand, we can change it to <command> <action> <name> format.
However for `update` command, id is the primary input, and name is optional param with which user can update the name as well, so we won't be able use <command> <action> <name> format.

So I think it's better we keep using the flag for name for consistent experience at least for all `policy` commands.